### DISCUSSION TITLE
Floating windows always on top?

### DISCUSSION THREAD

**Discussion by Okanda** [pop-os/shell#1672] [General]
Hello, 

is there a way to have the floating windows always on top like in others WM (Sway or Hyprland for exemple). 
I don't see any option about that. 

Thank you.

**Comment by damien-bafile** [comment]
You can open the always-on-top window twice and then just use one to select the other. I had the same issue for a while then i work it out by chance

**Reply by Okanda** [reply]
Not sure to understand. 
What I'm looking for is something automatic for all the floating windows (windows and app saved in the "Floating Windows Exceptions menu").

**Comment by 4x8Matrix** [comment]
Was there ever a solution to this? Running into an issue where my windows are falling behind my tiled windows..

**Reply by mmstick** [reply]
All development went to COSMIC, so you should think about switching to that if you want this.

**Reply by MendyBerger** [reply]
How is this done in COSMIC? I don't see such an option in the floating-windows exceptions screen.
(FWIW I'm using COSMIC on Wayland)

**Reply by mmstick** [reply]
There is no floating exceptions screen, unless you're referring to GNOME.
COSMIC has this behavior built in by default.
